content/docs/developer/anteaterapi/migrating.mdx

```diff
@@ -0,0 +1,95 @@
+* The `code` field/parameter has been renamed to `sectionCode` and is now a string containing the section code.
+
+### WebSoc (REST)
+* The path to this resource is now `/websoc` instead of `/schedule/soc`.
+* The `term` parameter has been replaced by the `year` and `quarter` parameters which together make up the two space-delimited parts of the `term` parameter. For example, if your original query used `term=2021%20Fall`, you should use `year=2021&quarter=Fall` instead.
+* The `meetings` and `finalExam` fields now contain additional metadata. See the [API Reference](https://anteaterapi.com/reference#tag/websoc/GET/v2/rest/websoc) for more details.
+
+## Breaking Changes Since v1
+
+This section applies to you if you are using the base URLs https://api-next.peterportal.org/v1/rest (for the REST API) or https://api-next.peterportal.org/v1/graphql (for the GraphQL API).
+
+### All REST endpoints
+* The top-level `statusCode`, `timestamp`, and `requestId` fields are no longer available.
+* The `success` and `payload` fields have been renamed to `ok` and `data` respectively.
+
+### Courses
+* The `/courses/all` endpoint is no longer available. It has been replaced by the `/courses` endpoint, which is paginated.
+* The `instructorHistory` field is no longer available. The data can still be accessed by taking the `id` field of each object in the `instructors` field.
+* The `shortenedName` field of the Instructor object is no longer available. It has been replaced by a `shortenedNames` field, which is an array containing each WebSoc name associated with that instructor.
+* The `prerequisiteList` and `prerequisiteFor` fields are no longer available. The data can still be accessed by taking the `id` field of each object in the `prerequisites` and `dependencies` fields, respectively.
+
+### Instructors
+* The `/instructors/all` endpoint is no longer available. It has been replaced by the `/instructors` endpoint, which is paginated.
+* The `shortenedName` field is no longer available. It has been replaced by a `shortenedNames` field, which is an array containing each WebSoc name associated with the instructor.
+* The `schools` and `relatedDepartments` fields are no longer available. At this time, there is no replacement for these fields.
+* The `courseHistory` field is no longer available. It has been consolidated into the `courses` field. Each object in the `courses` field now contains a `terms` field, which contains all terms in which the course was or will be taught.
+
+### Grades
+* If a section has only Pass and/or Not Pass grades, the `averageGPA` field will be `null`, rather than `0`.
+
+### WebSoc
+* The `cache` option is no longer available. In the interest of preventing WebSoc (the service managed by the school) from getting too many requests, at this time there is no replacement for the functionality derived from setting `cache` to `false`. To remedy this, objects at most levels have an `updatedAt` field to indicate when WebSoc was last fetched.
```

Comment:
"Previously this option proxied requests directly to WebSoc. If this functional is critical for your application, consider hosting your own proxy and use [our package] to normalize the data."